Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,6 +25,10 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
     "runtimeConfig": {
         "gcServer": true,
         "gcConcurrent": false
+    },
+    "targetFramework": {
```

Comment:
@davidfowl Where do you expect to find the SharedFramework name and version if not in runtime.config.json? Also, I hope we are not setting serverGC to true by default, right?